src/cloud/project/services-redis.md

```diff
@@ -5,12 +5,12 @@ functional_areas:
   - Cloud
   - Setup
 redirect_from:
-  - /cloud/project/project-conf-files_services-redis.html 
+  - /cloud/project/project-conf-files_services-redis.html
 ---
 
 [Redis](http://redis.io) is an optional, backend cache solution that replaces the Zend Framework [Zend_Cache_Backend_File](http://framework.zend.com/apidoc/1.0/Zend_Cache/Backend/Zend_Cache_Backend_File.html), which is used in Magento 2 by default.
 
-We support Redis versions 3.2 and 5.0. See [Configure Redis]({{ site.baseurl }}/guides/v2.3/config-guide/redis/config-redis.html).
```

Comment:
Here we write that we support Redis 3.2, 5.0, and 6.0. but on page https://devdocs.magento.com/cloud/project/services.html#service-versions exist version 4.0. It doesn't look clear for me.

Reply:
@andriyShevtsov Good catch. From here https://wiki.corp.magento.com/display/PRODM/Platform+Health+Roadmap it looks like 4.0 should be removed. So I will remove 4.0 from the service version table. If this is not correct, let me know.

Reply:
@oshmyheliuk and @andriyShevtsov: Redis 4.0 is also included in the docker container table, as shown here: https://devdocs.magento.com/cloud/docker/docker-containers.html#service-configuration-options.

So do I just need to add 4.0 to the `services-redis` page? It's unclear to me.